### DISCUSSION TITLE
我无法用PCL启动枪战服务器(物骨服)的客户端

### DISCUSSION THREAD

**Discussion by 0LYT0** [Meloong-Git/PCL#4799] [提问] [answered]
我无法用PCL启动枪战服务器(物骨服)的客户端，作者可以解释一下吗？我用HMCL是可以启动的，总之不是我电脑和网络的问题

**Comment by 0LYT0** [comment]
我希望作者可以了解一下我这边的问题

**Comment by lactobionicAcid** [comment]
请上传你的 PCL 启动器日志。

_另外，[物骨服好像无限期停服了](https://t.bilibili.com/948710202156253304)，只有客户端好像没什么用的说......_

**Comment by 0LYT0** [comment]
6

**Comment by 3gf8jv4dv** [comment]
Just saying *Cannot launch* and *6* does not mean anything. If you encounter a problem that prevents the game from launching, at least describe the specific circumstances of the failure to launch and upload the corresponding launcher log and game crash report.

只说一句 “无法启动” 和 “6” 并没有任何意义。如果遇到了此类问题，至少要描述一下无法启动的具体情况，并上传对应的启动器日志和游戏崩溃报告。

**Comment by LTCatt** [comment, marked as answer]
https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+Bug%2C%E6%96%B0%E5%8F%8D%E9%A6%88&projects=&template=bug1.yml
